lib/karafka/instrumentation/listener.rb

```diff
@@ -9,16 +9,15 @@ class << self
         # Logs details about each received message value parsing
         # @param event [Dry::Events::Event] event details including payload
         def on_params_params_parse(event)
-          topic = event[:caller].topic
-          time = event[:time]
-          debug "Params parsing for #{topic} topic successful in #{time} ms"
+          # Keep in mind, that a caller here is a params object not a controller, so it returns
+          # a topic as a string, not a routing topic
```

Comment:
Keep in mind, that a caller here is a param object not a controller, so it returns a topic as a string, not a routing topic